integrations/sofy_integration_demo_project/codemagic.yaml

```diff
@@ -0,0 +1,197 @@
+        - app_store_credentials # <-- (Includes APP_STORE_CONNECT_ISSUER_ID, APP_STORE_CONNECT_KEY_IDENTIFIER, APP_STORE_CONNECT_PRIVATE_KEY)
+          # APP_STORE_CONNECT_ISSUER_ID <-- Put your App Store Connect Issuer Id here 
+          # APP_STORE_CONNECT_KEY_IDENTIFIER <-- Put your App Store Connect Key Identifier here 
+          # APP_STORE_CONNECT_PRIVATE_KEY <-- Put your App Store Connect Private Key here
+        - certificate_credentials # <-- (Includes CERTIFICATE_PRIVATE_KEY - Put your Certificate Private Key here)
+        - other
+      vars:
+        XCODE_WORKSPACE: "Runner.xcworkspace"
+        XCODE_SCHEME: "Runner"                
+        BUNDLE_ID: "io.codemagic.flutteryaml" # <-- Put your bundle id here
+        APP_STORE_ID: 1111111111 # <-- Use the TestFlight Apple id number (An automatically generated ID assigned to your app) found under General > App Information > Apple ID. 
+      flutter: stable
+      xcode: latest
+      cocoapods: default
+    scripts:
+      - name: Set up keychain to be used for codesigning using Codemagic CLI 'keychain' command
+        script: |
+          keychain initialize
+      - name: Fetch signing files
+        script: |
+          app-store-connect fetch-signing-files $BUNDLE_ID --type IOS_APP_STORE --create
+      - name: Use system default keychain
+        script: |
+          keychain add-certificates
+      - name: Set up code signing settings on Xcode project
+        script: |
+          xcode-project use-profiles
+      - name: Get Flutter packages
+        script: |
+          flutter packages pub get
+      - name: Flutter analyze
+        script: |
+          flutter analyze
+      - name: Flutter unit tests
+        script: |
+          flutter test
+        ignore_failure: true          
+      - name: Install pods
+        script: |
+          find . -name "Podfile" -execdir pod install \;
+      - name: Flutter build ipa and automatic versioning
+        script: |
+          flutter build ipa --release \
+          --build-name=1.0.0 \
+          --build-number=$(($(app-store-connect get-latest-testflight-build-number "$APP_STORE_ID") + 1)) \
+          --export-options-plist=/Users/builder/export_options.plist
+      - name: Publish IPA to Sofy
+        script: |
+            curl --location --request POST 'https://api.sofy.ai/api/AppTests/buildUpload' \
+            --header "SubscriptionKey: $SOFY_SUBSCRIPTION_KEY" \
+            --form "applicationFile=@"$APK_PATH""
+      - name: Schedule an automation test with Sofy
+        script: |
+           curl --location --request POST 'https://api.sofy.ai/api/CICD/ScheduleAutomatedTestRun' 
+            --header 'Content-Type: application/json'    
+            --data-raw '{
+            "APIKey":"$SOFY_API_KEY",
+            "ScheduledID":9999
+            }'
+      - name: Schedule an automation test with Sofy
```

Comment:
The name of this script step is the same as the previous one. I see that it uses a different endpoint, so rename the s cript to show that something different is being done.